src/main/webapp/.eslintrc.js

```diff
@@ -48,5 +48,7 @@ module.exports = {
         endOfLine: "auto",
       },
     ],
+    "no-empty-function": "off",
+    "@typescript-eslint/no-empty-function": "off",
```

Comment:
Please remove these when you are done.

Reply:
Removed in 445c58cec70fbfbbe6113426d30eb0caf860e90f